src/Apps/Order/Components/ShippingAndPaymentDetails.tsx

```diff
@@ -51,38 +55,37 @@ export const ShippingAndPaymentDetails: React.SFC<
   ...others
 }) => (
   <Flex flexDirection="column" {...others}>
-    <StackableBorderBox flexDirection="column">
-      {fulfillmentType === "PICKUP" ? (
-        <>
-          <Serif size="3" weight="semibold" color="black100">
-            Pick up ({lineItems.edges[0].node.artwork.shippingOrigin})
-          </Serif>
-          <Serif size="3t" color="black100">
-            You’ll be appointed an Artsy specialist within 2 business days to
-            handle pickup logistics.
-          </Serif>
-        </>
-      ) : (
-        <>
-          <Serif size="3" weight="semibold" color="black100">
-            Ship to
-          </Serif>
-          <Serif size="3t" color="black100" style={{ whiteSpace: "pre-wrap" }}>
-            {renderAddress(addressProps)}
-          </Serif>
-        </>
-      )}
-    </StackableBorderBox>
-    <StackableBorderBox alignItems="center">
-      <CreditCardIcon
-        type={brand}
-        style={{ marginRight: space(1), position: "relative", top: "-2px" }}
-      />
-      <Serif size="3" color="black100">
-        •••• {last_digits}
-        &nbsp; Exp {expiration_month}/{expiration_year}
-      </Serif>
-    </StackableBorderBox>
+    {fulfillmentType === "PICKUP" ? (
+      <StepSummaryItem
+        onChange={onShippingChange}
+        title={<>Pick up ({lineItems.edges[0].node.artwork.shippingOrigin})</>}
+      >
+        You’ll be appointed an Artsy specialist within 2 business days to handle
+        pickup logistics.
+      </StepSummaryItem>
+    ) : (
+      <StepSummaryItem onChange={onShippingChange} title="Ship to">
+        <div style={{ whiteSpace: "pre-wrap" }}>
+          {renderAddress(addressProps)}
+        </div>
+      </StepSummaryItem>
+    )}
+    <StepSummaryItem
+      onChange={onPaymentChange}
+      title={onPaymentChange ? "Payment method" : null}
```

Comment:
It feels a little weird to infer whether or not to show the title based on the presence of `onPaymentChange`. Maybe we can have a separate prop for hiding the title so we are explicit? `hidePaymentTitle` or something like that?